pkg/customresourcestate/registry_factory.go

```diff
@@ -500,6 +545,13 @@ func getNum(value interface{}) (float64, error) {
 		}
 		return 0, nil
 	case string:
+		v := value.(string)
+		switch v {
+		case "True", "true":
+			return 1, nil
+		case "False", "false":
+			return 0, nil
+		}
```

Comment:
Maybe to be removed again.

It came from the `status.conditions` use case but there is also `Unknown` which is why this is more a statesSet instead.